core/app/(user)/reset/ResetForm.tsx

```diff
@@ -84,11 +88,9 @@ export default function ResetForm() {
 				<DialogContent>
 					<DialogHeader>
 						<DialogTitle>Success</DialogTitle>
-						<DialogDescription>
-							<p className="flex flex-col gap-2">
-								<span className="text-green-700">Success - password reset!</span>
-								Redirecting in 5 seconds...
-							</p>
+						<DialogDescription className="flex flex-col gap-2">
+							<span className="text-green-700">Success - password reset!</span>
+							Redirecting in 5 seconds...
 						</DialogDescription>
```

Comment:
`<DialogDescription>` already renders a `<p>`, so this was giving hydration errors bc you should not render `<p><p/></p>`